examples/custom-logback/values.yaml

```diff
@@ -2,3 +2,10 @@ logging:
   logback:
     existingConfigmap: "graphdb-custom-logback-config"
     configmapKey: "logback.xml"
+
+# You can use the same configuration for the proxy as well
```

Comment:
m?

Reply:
The same Logback configmap can be applied to the proxy too.